README.md

```diff
@@ -32,6 +32,35 @@
 - 2018-08-13: **地学 GMT 中文社区 2 群** QQ 群建立（群号：791856541）
 - 2019-01-27: [@seisman][seisman] 将模块手册和示例整理到中文手册中，废弃 modules 和 examples 子域名
 
+## DNS 信息
+
+DNS 使用[腾讯云](https://cloud.tencent.com/)进行解析。
+
+- 账户所有者: [@seisman](seisman)
+
+具体操作：
+
+1. 到 Godaddy 的“我的域名”->“域名设置”下，找到“管理DNS”
+2. 修改域名服务器为 `f1g1ns1.dnspod.net` 和 `f1g1ns2.dnspod.net`
+3. 前往腾讯云，添加域名 `gmt-china.org`
+4. 添加 `A` 记录，指定 GitHub Pages IP 地址 `185.199.108.153`
+5. 增加 CNAME 记录，二级域名 `docs.gmt-china.org` 指向 `gmt-china.github.io`
+6. 为使用腾讯企业邮箱，需增加如下记录
+
+   ```
+   @  MX   mxbiz1.qq.com  5
+   @  MX   mxbiz2.qq.com  10
+   ```
+
+DNS 记录如下图所示:
+
+![](https://user-images.githubusercontent.com/3974108/123645067-fd33fe80-d7f3-11eb-8aad-05b898981e52.png)
+
+生成静态网站并将网站托管到 GitHub Pages 时，还需要注意如下两点：
+
+1. 要有一个空文件 `.nojekyll` 以免 GitHub 使用 Jekyll 对网站再次渲染
```

Comment:
We don't have `.nojekall` files in docs or gmt-china.org.

Reply:
https://github.com/gmt-china/GMT_docs/tree/gh-pages

Reply:
docs use `deploy` to generate `.nojekall`, but how gmt-china.org create this file?

Reply:
Maybe.